UM/Qt/qml/UM/CheckBox.qml

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Uranium is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+
+CheckBox
+{
+    id: control
+
+    hoverEnabled: true
+
+    indicator: Rectangle
+    {
+        height: UM.Theme.getSize("checkbox").height
+        width: UM.Theme.getSize("checkbox").width
```

Comment:
```suggestion
        implicitWidth: UM.Theme.getSize("checkbox").width
```
If you're creating re-usable components, it's generally better to set the implicit width / height. Impicit height/widht is the width/height that is used if whoever creates the item doesn't set a size (So you can see it as "default size")